src/components/pages/Profile/History/VerifyForm.tsx

```diff
@@ -0,0 +1,125 @@
+import { Logger } from '@oceanprotocol/lib'
+import axios from 'axios'
+import { Field, Form, useFormikContext } from 'formik'
+import { graphql, useStaticQuery } from 'gatsby'
+import React, { ChangeEvent, ReactElement, useEffect, useState } from 'react'
+import { FormContent, FormFieldProps } from '../../../../@types/Form'
+import {
+  RegistryApiResponse,
+  VpDataBody
+} from '../../../../@types/Verification'
+import { useSiteMetadata } from '../../../../hooks/useSiteMetadata'
+
+import { useWeb3 } from '../../../../providers/Web3'
+import Button from '../../../atoms/Button'
+import Input from '../../../atoms/Input'
+import styles from './Verify.module.css'
+
+const query = graphql`
+  query {
+    content: file(relativePath: { eq: "pages/history.json" }) {
+      childPagesJson {
+        verify {
+          data {
+            name
+            type
+            placeholder
+            label
+            help
+            required
+          }
+        }
+      }
+    }
+  }
+`
+
+export default function VerifyForm({
+  accountIdentifier
+}: {
+  accountIdentifier: string
+}): ReactElement {
+  const data = useStaticQuery(query)
+  const content: FormContent = data.content.childPagesJson.verify
+
+  const { vpRegistryUri } = useSiteMetadata().appConfig
+
+  const {
+    isValid,
+    setErrors,
+    setFieldValue,
+    setStatus,
+    setTouched,
+    status,
+    validateField
+  } = useFormikContext()
+
+  useEffect(() => {
+    setErrors({})
+    setTouched({})
+
+    // setSubmitting(false)
+  }, [setErrors, setTouched])
+
+  const { accountId } = useWeb3()
+  const [addOrUpdate, setAddOrUpdate] = useState('Add')
+  useEffect(() => {
+    const initFileData = async () => {
+      try {
+        const response: RegistryApiResponse<VpDataBody> = await axios.get(
+          `${vpRegistryUri}/vp/${accountId}`
+        )
```

Comment:
issue(non-blocking): let's go with this approach for now but in the long run we should avoid this kind of call and use a facade pattern here. This allows us to switch implementation from Axios to any other library if we should ever decide to change it. Take this as an architecture improvement hint for our future projects.